templates/pt/rectification-default.txt

```diff
@@ -0,0 +1,17 @@
+A quem possa interessar:
+
+Solicitando a retificação dos meus dados pessoais incorretos, de acordo com o Artigo 16 do RGPD.
+
+As seguintes alterações devem ser feitas:
+{rectification_data}
+Caso você tenha divulgado os dados pessoais incorretos a terceiros, deverá informar minha solicitação de retificação a todos os destinatários, conforme estabelecido no Artigo 19 do RGPD. Por favor, também me informe sobre esses destinatários.
+
+Minha solicitação inclui explicitamente [runs > the following as well as] e quaisquer outros serviços e empresas para os quais você é o responsável, conforme definido no artigo 4 (7) do GDPR [runs>: {runs_list}].

```

Comment:
This one is not very clear, sorry.

The `the following as well as` needs to be translated, as well.

Then, please have no spaces around the `>`, i.e.: `[runs>your translation here]`.

Reply:
Sorry, I thought it wasn't necessary.

Reply:
No worries!